### PR TITLE
LP-2433 Integrate msp_dashboard with edx-platform.

### PR DESCRIPTION
**Pull request by qasimgulzar**
### Description
[LP-2433](https://philanthropyu.atlassian.net/browse/LP-2433)

### CODE REVIEW

**Comment by philanthropyu** [comment]
Your PR has finished running tests. The following contexts failed:
* jenkins/python

**Comment by philanthropyu** [comment]
Your PR has finished running tests. The following contexts failed:
* jenkins/python

File: lms/envs/common.py [hunk 1]
```diff
@@ -2409,7 +2409,8 @@ def _make_locale_paths(settings):
     'multiselectfield',
 
     # Third party action planner xblock goals tracking application
-    'action_planner.actionplan_manager'
+    'action_planner.actionplan_manager',
+    'msp_assessment.msp_dashboard',
```
Comment (ahmed-faraz46) [inline lms/envs/common.py:2413]: It looks like under the heading of `# Third party action planner xblock goals tracking application`. How about adding a separate heading for it or just adding separator.

**Review by ahmed-faraz46 — CHANGES_REQUESTED** [review]

**Review by saad-waqar — APPROVED** [review]

**Comment by philanthropyu** [comment]
Your PR has finished running tests. The following contexts failed:
* jenkins/python

**Review by ahmed-faraz46 — APPROVED** [review]
LGTM 👍

**Pull request merged**